### PR TITLE
Add Scheduler

### PR DESCRIPTION
**Pull request by arcbtc**

### CODE REVIEW

**Comment by arcbtc** [comment]
Issue with logo, I think its the size

**Comment by bitkarrot** [comment]
@dni @arcbtc bump.

**Comment by bitkarrot** [comment]
updated Scheduler to fix the install error. asking people to test it now on 0.12.10 and 0.12.11

will also check the updates @dni made to decoder and mirror to scheduler

File: extensions.json [hunk 1]
```diff
@@ -894,6 +894,17 @@
             "icon": "https://raw.githubusercontent.com/lnbits/lncalendar/main/static/image/lncalendar.png",
             "archive": "https://github.com/lnbits/lncalendar/archive/refs/tags/v0.1.2.zip",
             "hash": "8ea76bfaeb32f19fc0e1b8e48b286936842d1afb1d8d49233e78865cf55a3a79"
+        },
+        {
+            "id": "scheduler",
+            "repo": "https://github.com/bitkarrot/scheduler",
+            "name": "Scheduler",
+            "version": "0.1.1",
```
Comment (bitkarrot) [inline extensions.json:902]: update to version v0.1.2-alpha-2

File: extensions.json [hunk 2]
```diff
@@ -894,6 +894,17 @@
...
+            "min_lnbits_version": "0.12.0",
```
Comment (bitkarrot) [inline extensions.json:903]: min version 0.12.10

File: extensions.json [hunk 3]
```diff
@@ -894,6 +894,17 @@
...
+            "short_description": "Generate Scheduled tasks via crontabs for lnbits",
+            "icon": "https://raw.githubusercontent.com/bitkarrot/scheduler/main/static/image/scheduler.png",
+            "archive": "https://github.com/bitkarrot/scheduler/archive/refs/tags/v0.1.1.zip",
+            "hash": "db7470dc51a4d8bf3c0e3da49d6dd37bd584f60302778682165ce359d327e06c"
```
Comment (bitkarrot) [inline extensions.json:907]: f02fab0c1b0d6a6f2909a47e2d1692ae8900b1f0d1e4aeb7782585f8a664d7c0

File: extensions.json [hunk 4]
```diff
@@ -894,6 +894,17 @@
...
+        },
+        {
+            "id": "scheduler",
+            "repo": "https://github.com/bitkarrot/scheduler",
+            "name": "Scheduler",
+            "version": "0.1.1",
+            "min_lnbits_version": "0.12.0",
+            "short_description": "Generate Scheduled tasks via crontabs for lnbits",
+            "icon": "https://raw.githubusercontent.com/bitkarrot/scheduler/main/static/image/scheduler.png",
+            "archive": "https://github.com/bitkarrot/scheduler/archive/refs/tags/v0.1.1.zip",
```
Comment (bitkarrot) [inline extensions.json:906]: https://github.com/bitkarrot/scheduler/archive/refs/tags/v0.1.2-alpha-2.zip

**Review by bitkarrot — COMMENTED** [review]
I can't seem to edit this PR so made some comments on what needs to be updated.

**Comment by bitkarrot** [comment]
@arcbtc

**Comment by prusnak** [comment]
updated to  0.1.2-alpha-2 like @bitkarrot suggested

**Review by prusnak — APPROVED** [review]

**Comment by bitkarrot** [comment]
LFG!!!

**Review by dni — CHANGES_REQUESTED** [review]
```"version": "0.1.2-alpha-2",```
why this weird version?

**Comment by bitkarrot** [comment]
the only thing that changed in there was min version lnbits. 

would you be feel better if i named it v0.1.3? we can make another release and just rename it. LMK

@dni

**Comment by dni** [comment]
> the only thing that changed in there was min version lnbits.
> 
> would you be feel better if i named it v0.1.3? we can make another release and just rename it. LMK
> 
> @dni

yes please :)

**Review by motorina0 — APPROVED** [review]

File: extensions.json [hunk 5]
```diff
@@ -1259,6 +1259,17 @@
...
+        {
+            "id": "scheduler",
+            "repo": "https://github.com/bitkarrot/scheduler",
+            "name": "Scheduler",
+            "version": "0.1.2-alpha-2",
+            "min_lnbits_version": "0.12.10",
+            "short_description": "Generate Scheduled tasks via crontabs for LNbits",
+            "icon": "https://raw.githubusercontent.com/bitkarrot/scheduler/main/static/image/scheduler.png",
+            "archive": "https://github.com/bitkarrot/scheduler/archive/refs/tags/v0.1.2-alpha-2.zip",
+            "hash": "f02fab0c1b0d6a6f2909a47e2d1692ae8900b1f0d1e4aeb7782585f8a664d7c0"
```
Comment (prusnak) [inline extensions.json:1272]:
```suggestion
            "hash": "8d152c00371f001ed9712be1b265f5ff9900066fc78707e79fec24302151bbde"
```

**Comment by prusnak** [comment]
I updated the PR to latest version 0.1.3 but now the jmeter test fails - can you please @motorina0 have a look?

**Comment by prusnak** [comment]
Thanks @motorina0, now all tests are green.

@dni will you give us a green review since your request to change the version has been addressed, so we can push this through?

**Review by dni — APPROVED** [review]

**Pull request merged**